lawlibrary/templates/peachjam/about.html

```diff
@@ -0,0 +1,41 @@
+{% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
+
+{% block title %} {% trans 'About' %}{% endblock %}
+
+{% block page-content %}
+<section class="py-5">
+  <div class="container">
+    <h1 class="mb-4">{% trans 'About the platform' %}</h1>
+    <p>
+      {% blocktrans %}
+      Company copy
```

Comment:
@longhotsummer or @NotoriousMBB, I just need copy about the organisation over here

Reply:
https://docs.google.com/document/d/1qm-_EmY6lPwZVV3MTuas5M3VnJIdNDRrO4IKRChr1po/edit

Reply:
added the copy